src/views/feedback/constants.ts

```diff
@@ -0,0 +1,17 @@
+import { ReplyInfoReq } from '@/types/feedback'
+
+// 回复标题
+const TITLE = '收到反馈'
+
+// 默认回复内容
+const DEFUALT_CONTENT = '感谢您的反馈！我们非常重视您提供的信息，我们会尽快处理并采取适当的措施。'
+
+// 快捷回复消息
+export const replyMsg = ref<ReplyInfoReq>({
```

Comment:
- replyMsg 不是响应式变量！把 ref 去掉；
- 常量全部大写，用 _ 分隔，这点我应该提过；
- 
```suggestion
export const DEFAULT_REPLY_MSG = {
```